### ISSUE TITLE
PROJEKT-5: Planerade störningar ska alltid vara lägsta kritikalitet

### ISSUE THREAD

**Issue by FredrikElliot** [studio27se/ehub#123]
En planerad störning ska alltid läggas in med lägsta kritikalitet

**Comment by FredrikElliot** [comment]
Frågor från oss:

- Ska det gå att ändra kritikalitet på något sätt? (Även om default är lägsta)
- Ska man "behöva" fylla i krit.frågorna?

@monaAdawi @johulls

**Comment by johulls** [comment]
Jag kan inte riktigt se när man skall ändra den till hög. Men det är väl lika bra att man kan ändra den. Nej jag tycker inte att man behöver svara på frågorna. https://github.com/MonaAdawi är du av annan åsikt.
//Johan

**Comment by MonaAdawi** [comment]
@FredrikElliot @johulls Jag tycker man ska kunna ändra kritikalitet även om Låg är default. Innan jag svarar på andra punkten vill jag veta om flödet måste förändras vid en planerad då..? Alltså hur skulle det se ut om man tex inte behöver fylla i kritikalitetsfrågorna vid planerad?

**Comment by FredrikElliot** [comment]
Ja, då behöver vi justera flödet. Helt enkelt hoppa direkt från att välja objekt till skärmen där man trycker på registrera. Sen borde man väl i sådana fall kunna gå in och "ändra" svaren på kritikalitetsfrågorna för att justera kritikaliteten när driftstörningen är skapad. Detsamma för webappen. Men det är ju såklart upp till er.
@MonaAdawi @johulls

**Comment by MonaAdawi** [comment]
@FredrikElliot den här måste vi diskutera mer intern innan vi gör ngt med tycker jag. 
@johulls jag vill förstå hur "viktigt" det är att anpassa flödet vid planerade. Och säkerställa att vi inte "bygger bort oss".
@FredrikElliot sätt "invänta kund" på denna.

**Comment by MonaAdawi** [comment]
@FredrikElliot tagga denna som: Projekt

**Comment by MonaAdawi** [comment]
@FredrikElliot kan du döpa om rubriken till Projekt-5:

**Comment by MonaAdawi** [comment]
@FredrikElliot Nu kunde jag för första gången testa och se hur "infotexten" i appen ser ut när jag laddade hem version 1.8.10.
Jag tycker infotexten "försvinner" lite med övrig text. Kan vi (precis som på webben) "belysa" den lite genom en färgplatta eller lila färg på texten eller liknande. 

**Webb:**

![Image](https://github.com/user-attachments/assets/43d4791d-528c-4a7d-8c5b-6f76d90219ab)

**APP:** 

![Image](https://github.com/user-attachments/assets/e8d44458-526b-490c-a800-aea4b14b5a09)

**Comment by FredrikElliot** [comment]
@monaadawi 
Det kan vi, men jag tycker inte vi ska hålla tillbaka den här uppdateringen så jag kommer i sådana fall vilja ta det i ett nytt issue. Ska jag skapa? Så kan vi stänga detta?

**Comment by MonaAdawi** [comment]
@FredrikElliot okej då gör vi så. Du kan skapa det.

**Comment by MonaAdawi** [comment]
@FredrikElliot vi kan stänga denna men det finns ju fortfarande kvar ett fel i när man ändrar störningsdetaljer i en Planerad aktiv störning. Alltså när man byter svar på frågorna så kan Planerade störningar skifta till Hög = fel, se issue 143 Men så länge vi har kvar det issue´t så kan vi ju stänga denna.
